container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -313,6 +316,9 @@ function run_rpm_tempest {
     tempest init openshift
     pushd $TEMPEST_DIR
 
+    # Install additional plugins from .rpms plus their dependencies
+    [ ${#TEMPEST_EXTRA_RPMS[@]} -ne 0 ] && sudo dnf install -y ${TEMPEST_EXTRA_RPMS[@]}
```

Comment:
i was unsure of that bash syntax but it is correct 

${#TEMPEST_EXTRA_RPMS[@]}

the # gives the number of elements